src/components/Questionnaire/QuestionTypes/ServiceRequestQuestion.tsx

```
@@ -0,0 +1,724 @@
+  return values.reduce((errors: QuestionValidationError[], value, index) => {
+    const fieldErrors = Object.entries(SERVICE_REQUEST_FIELDS)
+      .filter(([_, field]) => field.required && !value[field.key])
+      .map(([_, field]) => ({
+        question_id: questionId,
+        error: "field_required",
+        type: "validation_error",
+        field_key: field.key,
+        index,
+      }));
+
+    return [...errors, ...fieldErrors];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)